contracts/access/manager/AccessManager.sol

```diff
@@ -18,7 +17,7 @@ interface IAccessManager is IAuthority, IAccessControlDefaultAdminRules {
 
     event GroupAllowed(address indexed target, bytes4 indexed selector, uint8 indexed group, bool allowed);
 
-    event AccessModeUpdated(address indexed target, AccessMode indexed mode);
+    event AccessModeUpdated(address indexed target, AccessMode indexed previousMode, AccessMode indexed mode);
```

Comment:
I don't think the previous one needs to be indexed. If you're looking up by mode you probably want to find the event that changed into a specific mode so you would look up by `mode`?